pinot-controller/src/main/java/com/linkedin/pinot/controller/api/restlet/resources/PinotSegmentUploadRestletResource.java

```diff
@@ -350,7 +351,12 @@ public Representation post(Representation entity) {
 
           // 3/ Request is parsed by the handler which generates a
           // list of FileItems
-          items = upload.parseRequest(getRequest());
+          try {
+            items = upload.parseRequest(getRequest());
+          } catch (FileUploadBase.IOFileUploadException e) {
```

Comment:
According to the EKG tickets, all of them throw a IOFileUploadException
2015/12/09 04:33:40 ERROR [PinotSegmentUploadRestletResource] [qtp111819772-648954] [pinot-controller] Caught exception in file upload
org.apache.commons.fileupload.FileUploadBase.IOFileUploadException: Processing of mul